src/main/java/org/jahia/modules/sam/healthcheck/probes/SearchIndexProbe.java

```diff
@@ -62,10 +62,10 @@ public ProbeStatus getStatus() {
         JahiaRepositoryImpl repository = (JahiaRepositoryImpl) ((SpringJackrabbitRepository) JCRSessionFactory.getInstance().getDefaultProvider().getRepository()).getRepository();
         RepositoryStatistics repositoryStatistics = repository.getContext().getRepositoryStatistics();
         double queryAVG = Arrays.stream(repositoryStatistics.getTimeSeries(RepositoryStatistics.Type.QUERY_AVERAGE).getValuePerMinute()).average().orElse(Double.NaN);
-        if (queryAVG > queryAVGLastMinuteYellowThreshold) {
-            return new ProbeStatus(yellowStatus.format(new Object[]{queryAVG, queryAVGLastMinuteYellowThreshold}), ProbeStatus.Health.YELLOW);
-        } else if (queryAVG > queryAVGLastMinuteRedThreshold) {
+        if (queryAVG > queryAVGLastMinuteRedThreshold || Double.isNaN(queryAVG)) {
```

Comment:
can we reverse the `||` order ? I think if it's nan, the > comparison will fail/throw error